### ISSUE TITLE
can't spawn multiple atlases

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#455]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/455)) by Jackie K (Bitbucket: [jacquelinekay](https://bitbucket.org/%7Be3d60b62-b7c7-4863-b9b9-cdae9f123cec%7D/)).**

----------------------------------------
Not a common use case for DRCSim, but spawning multiple Atlases using the "Insert Model" tab causes Gazebo to crash:
```
terminate called after throwing an instance of 'Ogre::ItemIdentityException'
  what():  OGRE EXCEPTION(4:ItemIdentityException): A camera with the name default::head_hokuyo_sensor_ortho_cam already exists in OctreeSceneManager::createCamera at /build/buildd/ogre-1.8-1.8.1+dfsg/PlugIns/OctreeSceneManager/src/OgreOctreeSceneManager.cpp (line 372)
```

**Comment by osrf-migration** [comment]
**Original comment by Jackie K (Bitbucket: [jacquelinekay](https://bitbucket.org/%7Be3d60b62-b7c7-4863-b9b9-cdae9f123cec%7D/)).**

----------------------------------------

* Edited issue description

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
I ran into this today when I wanted to spawn multiple versions of atlas side-by-side to compare their kinematic differences. It's worth noting that the [head_hokuyo_sensor](https://github.com/osrf/drcsim/blob/bff7f672257b53d38c6dea6a7ed2d20513890567/multisense_sl_description/urdf/multisense_sl_v4.urdf#L165) is a `gpu_ray` sensor.

@iche033 is this a known issue with the `gpu_ray` sensor?

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
I'm going to create a gazebo issue for this.

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
[gazebo issue 1403](https://bitbucket.org/osrf/gazebo/issue/1403/spawning-multiple-gpu_ray-sensors-with)

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
I just tested a fix. We will make a gazebo pull request soon.

![atlas evolution.png](https://osrf-migration.github.io/drcsim-gh-pages/data/bitbucket.org/repo/bae8oa/images/3393357641-atlas_evolution.png)

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
https://bitbucket.org/osrf/gazebo/pull-request/1360/handle-gpulaser-name-collisions-1403/diff

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------
It is fixed in gazebo if you build from source, but it hasn't been released yet.

**Comment by osrf-migration** [comment]
**Original comment by Jackie K (Bitbucket: [jacquelinekay](https://bitbucket.org/%7Be3d60b62-b7c7-4863-b9b9-cdae9f123cec%7D/)).**

----------------------------------------
Is the protocol to close this issue after Gazebo 4.1 gets patched?

**Comment by osrf-migration** [comment]
**Original comment by Steve Peters (Bitbucket: [Steven Peters](https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/), GitHub: [scpeters](https://github.com/scpeters)).**

----------------------------------------

* changed state from "new" to "resolved"

issue resolved in gazebo 4.1.1, which has been released
